README.md

```
@@ -1,185 +1,116 @@
-- [x] 선택한 상품이 없으면 주문할 수 없다.
-
-#### 최근 본 상품 기반 추천
-
-- [x] 사용자가 최근 본 상품의 카테고리를 저장한다.
-- [x] 가장 최근에 본 상품의 카테고리를 기준으로 추천 상품을 조회한다.
-- [x] 같은 카테고리 상품을 최대 10개까지 노출한다.
-- [x] 이미 장바구니에 담긴 상품은 추천 목록에서 제외한다.
-
-#### 추천 상품 장바구니 추가
+### 1. 쿠폰
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)